components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -98,58 +52,34 @@ export type InfoListItemProps = {
 };
 
 export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
-    const theme = useTheme();
-    const defaultClasses = useStyles(theme);
+    const defaultClasses = useStyles(props);
     const {
         avatar,
-        backgroundColor,
         chevron,
         classes,
         dense,
         divider,
-        fontColor,
         hidePadding,
         icon,
-        iconColor,
         leftComponent,
         onClick,
         rightComponent,
-        statusColor,
         style,
         subtitle,
         subtitleSeparator,
         title,
         ripple,
     } = props;
 
-    const getIconColor = (): string => {
-        if (iconColor) return iconColor;
-        if (avatar) {
-            return statusColor
-                ? color(statusColor).isDark()
-                    ? Colors.white[50]
-                    : Colors.black[500]
-                : Colors.white[50]; // default avatar is dark gray -> white text
-        }
-        return statusColor ? statusColor : 'inherit';
-    };
-
     const getIcon = (): JSX.Element | undefined => {
         if (icon && avatar) {
             return (
                 <ListItemAvatar>
-                    <Avatar
-                        style={{
-                            backgroundColor: statusColor || Colors.black[500],
-                            color: getIconColor(),
-                        }}
-                    >
-                        {icon}
-                    </Avatar>
+                    <Avatar className={defaultClasses.avatar}>{icon}</Avatar>
                 </ListItemAvatar>
             );
         } else if (icon) {
-            return <ListItemIcon style={{ color: getIconColor() }}>{icon}</ListItemIcon>;
+            return <ListItemIcon className={defaultClasses.icon}>{icon}</ListItemIcon>;
```

Comment:
can/should we expose this in classes prop for customization?

Reply:
I think so, I added it in.